stories/plotxy.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { ComponentStory, Meta } from '@storybook/react';
+import React from 'react';
+
+import data from '../src/data/variables.json';
+import PlotXY, {
+  Data,
+  DataXML,
+  Margins,
+  XmlPlotXY,
+} from '../src/layout/PlotXY';
+import LineSerie from '../src/layout/PlotXY/components/LineSerie';
+import XYAxis from '../src/layout/PlotXY/components/axis/xy-axis';
+
+export default {
+  title: 'Layout/PlotXY',
```

Comment:
You can use the `component` property so that storybooks generates the args from the types. You might want to give it a try.

Reply:
Do you have a code exmple please

Reply:
https://storybook.js.org/docs/react/api/csf#default-export

Here it should be `component: PlotXY`

Reply:
done